### DISCUSSION TITLE
<title>Bem vindo ao Perfil de Feedback - BR

### DISCUSSION THREAD

**Discussion by Gill3s0x01** [community/community#40008] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Deixem sua opinião e sugestões quanto ao serviço.

**Comment by ossanna16** [comment]
Thanks for participating!
We take our Code of Conduct very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. What this means is that, until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
